pkgs/misc/cups/drivers/mfcl2700dncupswrapper/default.nix

```diff
@@ -0,0 +1,44 @@
+{ coreutils, dpkg, fetchurl, gnugrep, gnused, makeWrapper, mfcl2700dnlpr, perl, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "mfcl2700dncupswrapper-${meta.version}";
+
+  src = fetchurl {
+    url = "http://download.brother.com/welcome/dlf102086/${name}.i386.deb";
+    sha256 = "07w48mah0xbv4h8vsh1qd5cd4b463bx8y6gc5x9pfgsxsy6h6da1";
+  };
+
+  nativeBuildInputs = [ dpkg makeWrapper ];
+
+  phases = [ "installPhase" ];
```

Comment:
@0xABAB the attribute `phases` is documented mentioning it can be used for skipping phases but I don't think its explicitly recommended to use it for skipping phases. The issue is that sometimes logic in some of the phases is needed but not executed because the phases are introduced. A common case is multiple outputs.